custom-payment-flow/client/react-native-expo/App.tsx

```diff
@@ -24,11 +27,38 @@ export default function App() {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
+  const handleDeepLink = useCallback(
+    async (url: string | null) => {
+      if (url && url.includes('safepay')) {
```

Comment:
This bit is interesting. I don't understand why we need to check for safepay. Is that always in the return URL?

Reply:
Yes, the Stripe SDK appends safepay, so that's what we can use to check whether we're redirecting from Stripe. It's not really needed though, as `handleURLCallback` simply does nothing when called with a non-stripe URL.

Reply:
got it. thanks!